packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
   - Display different pricing for various products on Product tiles and PDP [#1760](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1760) 
   - Display pricing for cart, checkout and wishlist page [#1796](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1796) 
   - Shows promotional callout message on Product List and Product Detail pages [#1786](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1786) [#1804](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1804)
-
+  - [a11y] Update live region for display price [#1825](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1825)
```

Comment:
NIT: Looking at this PR we are applying updates for aria live support to multiple components, not just the <DisplayPrice/> component. If the list isn't too long I'd also add them in this comment, otherwise simple add something like "added live region suppose to components"